src/components/home/MapSection.js

```diff
@@ -3,23 +3,26 @@ import { Grid, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/styles';
 
 import { useStylesBase } from '../styledComponents';
-import map from '../../assets/homePage/map.jpg';
+import map from '../../assets/homePage/map.png';
 
 const useStyles = makeStyles({
-  mainContainer: {
-    height: '680px',
-    backgroundImage: `url(${map})`,
+  title2: { margin: '100px 0px -250px' },
+  image: {
+    position: 'relative',
+    width: 'calc(100vw - 8px)',
+    left: 'calc(-1 * (100vw - 100%) / 2)', // to centered the div
```

Comment:
Didn't we change this in the last PR?

Reply:
No it is not the same this is for the map image (yes it is a copy paste)